providers/catalog.go

```diff
@@ -1711,4 +1712,31 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+	// InstagramBasic Configuration
+	InstagramBasic: {
```

Comment:
Can you call this just `Instagram`?

Reply:
Can you also add a comment above this - `TODO: Supports only short-lived tokens.`?

Reply:
updated